src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlClientLogger.cs

```diff
@@ -33,8 +41,5 @@ public bool LogAssert(bool value, string type, string method, string message)
                 LogError(type, method, message);
             return value;
         }
-
-        /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlClientLogger.xml' path='docs/members[@name="SqlClientLogger"]/IsLoggingEnabled/*'/>
-        public bool IsLoggingEnabled => Bid.TraceOn;
```

Comment:
Do not remove public API - return `Log.IsTraceEnabled()`

Reply:
It was a mistake happening in early stage of implementation. It is back. thanks for catching this.